### PR TITLE
Fix feature flaws

### PR DESCRIPTION
**Pull request by gohyongjing**
### Fixes
* Update shadowjar output file name to `friendlylink.jar`
* Remove addtional space in command feedback for elderly and volunteer related commands
* Standardise unspecified fields to empty string
  * Remove negative address tests since address is now always valid
* Change `START_DATE` to `AVAILABLE_DATE_START` and `END_DATE` to `AVAILABLE_DATE_END` in command usage description for consistency
* Update UI image in `userGuide.md`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ These terms have specific meanings in the context of FriendlyLink. For a more de
     * Open the app by entering `java -jar friendlylink.jar` into the terminal.
    
    The GUI should appear in a few seconds.
-   ![Ui](images/Ui.png)
+   ![Ui](images/readMe/Ui.png)
```
Comment (Zxun2) [inline docs/UserGuide.md:197]: As in removing the current `Ui.png`, and replacing it with the one you have.

**Review by Zxun2 — APPROVED** [review]
Yup, looks good. Are you suppose the update the autocomplete suggestions too?

File: docs/UserGuide.md (hunk 1 above)
Comment (gohyongjing) [reply docs/UserGuide.md]: Fixed in [993815d](https://github.com/AY2223S2-CS2103T-W12-1/tp/pull/193/commits/993815d6b488787fe551c84fac284e22bc1d6cbc)

**Comment by gohyongjing** [comment]
I believe autocomplete is done by someone else

**Pull request merged**